lib/fog/metering/openstack/requests/get_samples.rb

```diff
@@ -2,7 +2,7 @@ module Fog
   module Metering
     class OpenStack
       class Real
-        def get_samples(meter_id, options = [])
+        def get_samples(meter_id, options = [], limit = 10000000)
```

Comment:
Since the `option` hash in our method becomes the `q` JSON hash which has a specific schema to query ceilometer. The `limit` param is entirely separate from that.

So there's likely client code out there that looks like this:

```ruby
timestamp_format = "%Y-%m-%dT%H:%M:%S"
options = [{'field' => 'timestamp', 'op' => 'ge', 'value' => from.utc.strftime(timestamp_format)},
                   {'field' => 'timestamp', 'op' => 'lt', 'value' => to.utc.strftime(timestamp_format)}]
Fog::Metering.new(params).metering.get_samples(measurement, options)
```

That array becomes the `q` contents in the post body. To stay backwards compatible I think the results limit has to be a separate optional arg.